js/cell.js

```diff
@@ -465,7 +370,11 @@ define([
     }
 
     return {
-        make: make,
+        make: function (args, cellInfo, k) {
```

Comment:
The reason I made a wrapper was to avoid a big diff with indentation change, but I'm happy to move `domReady` inside of `_make` and rename it back to `make`.

The one weird point is that `_make` calls itself recursively. Of course `domReady` will immediately execute if the dom is ready, but it makes the recursive call a little less "pure".